### PR TITLE
Multiple text edits

### PR DESCRIPTION
**Pull request by meizabel**
Multiple text edits based on previous discussions.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -176,13 +176,13 @@ <h1 class="light section-title">Why Spine</h1>
                     <i class="fas fa-lightbulb-on card-icon-font"></i>
                     <div class="card-content-text">
                         <header class="feature-card-title">Clear API</header>
-                        <p>Concepts known from the DDD books right are right in the code.
-                            <code><a href="https://spine.io/core-java/javadoc/server/io/spine/server/aggregate/Aggregate.html">Aggregate</a></code>,
+                        <p>Concepts from the DDD books, such as
+                            <code><a href="hOpen for Business Changesttps://spine.io/core-java/javadoc/server/io/spine/server/aggregate/Aggregate.html">Aggregate</a></code>,
```
Comment (alexander-yevsyukov) [inline index.html]: I think the text after `h` was intended to somewhere else.

File: about/index.html [hunk 2]
```diff
@@ -92,7 +91,7 @@ <h1 class="light section-title">Features</h1>
                     <div class="card-content-text">
                         <header class="feature-card-title">Open Source</header>
                         <p>Permissive Apache License allows using the framework in closed-source
-                            projects. You are welcome to contribute to the framework development.</p>
+                            projects. You are also welcome to contribute to our framework development!</p>
```
Comment (alexander-yevsyukov) [inline about/index.html]: Why do we want the exclamation in this sentence?

File: index.html [hunk 3]
```diff
@@ -54,8 +55,8 @@ <h2 class="landing-row-title">Define rich, type&#8209;safe domain&nbsp;model</h2
             <div class="col-md-6 col-style-text">
                 <p class="step-title">Step 2.</p>
                 <h2>Generate the code for all tiers and multiple languages</h2>
-              <p>Java, JavaScript, and C++ are currently supported, with more languages to come soon.</p>
-            </div>
+              <p>Java, JavaScript, and C++ are currently supported. </p>
+                <!--Add a link to Roadmap for more languages/technologies support when available. --></div>
```
Comment (alexander-yevsyukov) [inline index.html]: Please remove the comment and have an issue.

File: index.html [hunk 4]
```diff
@@ -2,7 +2,8 @@
 layout: landing
 title: Spine Event Engine
 ---
-
+<!--Todo: add a reworked product description. Current proposal to be discussed with PR and marketing team: "Apply reactive Domain-Driven Design quicker, with less code transforming EventStorming into a domain model code.
```
Comment (alexander-yevsyukov) [inline index.html]: Please remove the comment and have an issue.

File: about/index.html [hunk 5]
```diff
@@ -113,8 +112,8 @@ <h1 class="light section-title">Features</h1>
                     <div class="card-content-text">
                         <header class="feature-card-title">Multitenancy Support</header>
                         <p>Transforming a single-tenant application into
-                            a&nbsp;multi-tenant one requires very few lines of code. You don't have to introduce
-                            <code>tenantId</code> parameters for all the calls.</p>
+                            a&nbsp;multi-tenant one requires a few lines of code. You don't have to introduce
+                            <code>tenantId</code> parameters for all the calls. <comment>Provide a link from this feature to gitHub repo or a page intended for contributors.</comment></p>
```
Comment (alexander-yevsyukov) [inline about/index.html]: Please remove the comment and have an issue.

File: index.html (hunk 1 above)
Comment (meizabel) [reply index.html]: LGTM.

File: about/index.html (hunk 2 above)
Comment (meizabel) [reply about/index.html]: Removed the exclamation mark at the end of the sentence.

File: index.html (hunk 3 above)
Comment (meizabel) [reply index.html]: Removed from source code. 
Converted into issue #7 in Documentation board. 
[For some reason cannot link it to https://github.com/SpineEventEngine/documentation/issues/7]

File: about/index.html (hunk 5 above)
Comment (meizabel) [reply about/index.html]: LGTM
Created issue #119

File: index.html (hunk 4 above)
Comment (meizabel) [reply index.html]: LGTM
Removed from the source code. 
Converted to documentation issue #8
[https://github.com/SpineEventEngine/documentation/issues/8]

**Comment by meizabel** [comment]
Issue #68 which references this one is to be addressed to in a separate PR.

**Pull request merged**